siliconcompiler/core.py

```diff
@@ -2342,21 +2330,22 @@ def _collect(self, step, index):
 
         '''
 
-        indir = os.path.join(self._getworkdir(step=step, index=index), 'inputs')
+        if not directory:
+            directory = os.path.join(self._getcollectdir())
```

Comment:
Is this preferable to accepting step/index in the `_collect` method, and:
```
if not directory:
    directory = os.path.join(self._getworkdir(step=step, index=index), 'inputs')
```
It looks like that might be the most common override for the `directory` kwarg, anyways.

Reply:
step/index doesn't really mean anything for the `_collect()` function. Before it was just copying files into each import step (the same files). So I don't think it makes sense to provide it as a `_collect`, and I don't think there are  any instances of using the directory override.

Reply:
It is used in the chisel driver, but fair enough.